datahub-web-react/src/graphql-mock/fixtures/browseDataset/datasetBrowseSnowflake.ts

```diff
@@ -0,0 +1,101 @@
+import { GetBrowseResults } from '../../types';
+import { findDatasetByPlatform } from '../searchResult/datasetSearchResult';
+
+const datasets = findDatasetByPlatform('snowflake');
+
+export const datasetBrowseProdSnowflake: GetBrowseResults = {
+    data: {
+        browse: {
+            entities: [],
+            start: 0,
+            count: 0,
+            total: 0,
+            metadata: {
+                path: ['prod', 'snowflake'],
+                groups: [
+                    { name: 'mydb', count: 3, __typename: 'BrowseResultGroup' },
+                    { name: 'novelty', count: 2, __typename: 'BrowseResultGroup' },
+                ],
+                totalNumEntities: 72,
+                __typename: 'BrowseResultMetadata',
+            },
+            __typename: 'BrowseResults',
+        },
+    },
+};
+
+export const datasetBrowseProdSnowflakeMydb: GetBrowseResults = {
+    data: {
+        browse: {
+            entities: [],
+            start: 0,
+            count: 0,
+            total: 0,
+            metadata: {
+                path: ['prod', 'snowflake', 'mydb'],
+                groups: [{ name: 'schema', count: 3, __typename: 'BrowseResultGroup' }],
+                totalNumEntities: 3,
+                __typename: 'BrowseResultMetadata',
+            },
+            __typename: 'BrowseResults',
+        },
+    },
+};
+
+export const datasetBrowseProdSnowflakeMydbSchema = ({ start }): GetBrowseResults => {
+    return {
+        data: {
+            browse: {
+                entities: datasets.slice(0, 3),
+                start,
+                count: 3,
+                total: 3,
+                metadata: {
+                    path: ['prod', 'snowflake', 'mydb', 'schema'],
+                    groups: [],
+                    totalNumEntities: 3,
+                    __typename: 'BrowseResultMetadata',
+                },
+                __typename: 'BrowseResults',
+            },
+        },
+    };
+};
+
+export const datasetBrowseProdSnowflakeNovelty: GetBrowseResults = {
+    data: {
+        browse: {
+            entities: [],
+            start: 0,
+            count: 0,
+            total: 0,
+            metadata: {
+                path: ['prod', 'snowflake', 'novelty'],
```

Comment:
again, we had some randomly generated datasets in the demo data but I found they caused more confusion than they helped. Let's use more realistic names in this sample data.